apps/admin-x-activitypub/src/views/Explore/Explore.tsx

```
@@ -0,0 +1,142 @@
+                {profile.actor.summary &&
+                    <div
+                        dangerouslySetInnerHTML={{__html: profile.actor.summary}}
+                        className='ap-profile-content mt-0 max-w-[500px]'
+                    />
+                }
+            </div>
+        </div>
+    );
+};
+
+const Profile: React.FC = () => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Profile'. Consider to delete it or rename it.

'Profile' is defined here:


(lint/suspicious/noRedeclare)